productlist.tex

```diff
@@ -0,0 +1,208 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%  Product table generated by makeProductTree.py do not modify.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\begin{longtable}{|p{0.1\textwidth}|p{0.2\textwidth}|p{0.4\textwidth}|p{0.15\textwidth}|p{0.15\textwidth}|}\hline 
```

Comment:
This table is too wide. Consider a sideways table. The Manager column is wider than it needs to be and what does a manager of "SuperTask" mean?